pkgs/by-name/la/latexminted/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "latexminted";
+  version = "0.1.0b1";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-O+p08Yg7seL3Nn4B+osV5EizLDfDTCh+YILRYgENPs4=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  pythonPath = with python3Packages; [
+    pygments
+    latex2pydata
+  ];
+
+  meta = {
+    homepage = "https://pypi.org/project/latexminted";
+    description = "Python executable for LaTeX minted package";
+    mainProgram = "latexminted";
+    license = lib.licenses.lppl13c;
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [ romildo ];
```

1. Ascii-betical ordering is nicer.
2. No need to set platforms

```suggestion
    license = lib.licenses.lppl13c;
    mainProgram = "latexminted";
    maintainers = with lib.maintainers; [ romildo ];
```

---

Done.